include/highfive/H5File.hpp

```diff
@@ -110,6 +110,12 @@ class File: public Object, public NodeTraits<File>, public AnnotateTraits<File>
         return details::get_plist<FileAccessProps>(*this, H5Fget_access_plist);
     }
 
+    /// \brief Get the disk size of this file in bytes
+    size_t getDiskSize() const;
```

Comment:
This is a wrapper for the function `H5Fget_filesize`. Could we use `getFileSize` instead?